Example/Example for Delirium/Modules/Dynamic Type/SelectFontFamilyViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SelectFontFamilyViewController.swift
+//  Delirium
+//
+//  Created by Nikki Vergracht on 22/05/2017.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+/// Example extension you could use in your app with your own fontFamily in order to not having to pass the fontFamily every time.
+extension UIFont {
+    
+    @nonobjc static var fontFamily = "Chalkboard SE" // Don't use this font family, ever.
+    
+    @available(iOS 7.0, *)
+    class func preferredCustomFont(fortextStyle style: UIFontTextStyle) -> UIFont {
```

Comment:
Same as above, the idea is to use the size of preferredFont as configured in the user's Settings. The font sizes are inferred for each specific TextStyle. More on the subject [here](https://developer.apple.com/ios/human-interface-guidelines/visual-design/typography/)! However, I'm open for discussion on this.